app/public/css/styles.css

```diff
@@ -1646,6 +1647,29 @@ button[data-balloon] {
   margin-right: 16px;
 }
 
+.buy-ada {
+  margin-top: 40px;
+  display: flex;
+  align-items: center;
+}
+
+.buy-ada-link,
+.buy-ada-link:hover {
+  text-decoration: none;
+}
+
+.buy-ada-logo {
+  margin-left: 24px;
+  font-size: 14px;
+  color: var(--color-grey-medium);
+  padding-bottom: 4px;
+}
+
+.buy-ada-icon {
```

Comment:
not sure what are good css naming conventions but this seems like it should be named `buy-ada-partner-logo` and the wrapping element (the one named `buy-ada-logo` now if I understand right) could be named `buy-ada-partner-logo-wrapper`

Reply:
okay, use `buy-ada-partner` as the main class name, then `buy-ada-partner-wrapper` etc.